### ISSUE TITLE
verifyBamID can't handle missing trailing fields of sample

### ISSUE THREAD

**Issue by matmu** [Griffan/VerifyBamID#13]
I just analyzed a VCF file created with GATK 3.8 pipeline and got the following error:

```
terminate called after throwing an instance of 'VcfFileException'
  what():  # values = ./.:0,0,0:.:.:0|1:18576120_T_C do not match with # fields in FORMAT field = 7 at sampleIndex = 7 See line 47626.
verifyBamID 1.0.0 -- verify identity and purity of sequence data
```

It turned out that this error occurs because GATK CombineGVCFs drops trailing fields per sample if they are empty. Apparently, this is allowed by the VCF format specification: [https://gatkforums.broadinstitute.org/gatk/discussion/4149/format-fields-and-sample-entries-in-vcf-files#latest](https://gatkforums.broadinstitute.org/gatk/discussion/4149/format-fields-and-sample-entries-in-vcf-files#latest)

Is there a parameter to ignore incomplete sample information?

**Comment by Griffan** [comment]
Yes, you are right. But at the moment we have not enough hands to amend this feature. You are more than welcome to help to contribute to this package. Otherwise, you can try to use a script to fill in the field in your specific vcf file.

We will try to fix this maybe later this semester.

**Comment by matmu** [comment]
I see the point. I am also quite busy right now, but I can extend the error message and link a script which adds the missing fields. Just have to find/create. Or do you know a tool that does this?

**Comment by Griffan** [comment]
Unfortunately no, if you finished this script, you can try to create a pull request. Thanks!
